administrator/components/com_media/controllers/file.json.php

```diff
@@ -172,7 +171,7 @@ public function upload()
 		}
 
 		// Trigger the onContentAfterSave event.
-		$dispatcher->trigger('onContentAfterSave', array('com_media.file', &$object_file, true));
+		$dispatcher->dispatch('onContentAfterSave', array('com_media.file', &$object_file, true));
```

Comment:
Yup. I missed this one. Fixed!

I also checked if we have any other stray calls to dispatch() and can't seem to find any more references.